eureka2-core/src/main/java/com/netflix/eureka2/channel/AbstractServiceChannel.java

```diff
@@ -76,6 +76,8 @@ protected boolean moveToState(STATE from, STATE to) {
                     metrics.incrementStateCounter(to);
```

Comment:
I have deliberately removed Idle state tracking, as it has unclear semantic. There was no connection attempt yet, just pure object construction. It is easy to put it back if we see value of it.